Plugins/Damage/NWScript/nwnx_damage.nss

```diff
@@ -26,7 +26,7 @@ struct NWNX_Damage_DamageEventData // Damage event data
 void NWNX_Damage_SetDamageEventScript(string sScript, object oOwner=OBJECT_INVALID);
 
 // Get Damage Event Data (to use only on Damage Event Script)
-struct NWNX_Damage_DamageEventData NWNX_Weapon_GetDamageEventData();
+struct NWNX_Damage_DamageEventData NWNX_Damage_GetDamageEventData();
```

Comment:
Can you add the old name as well, that just prints a warning and calls the new one?